package/MDAnalysis/guesser/base.py

```diff
@@ -0,0 +1,48 @@
+class GuesserBase:
+    context = "base"
+    _guess = {}
+
+    def is_guessed(self, to_guess):
+        """check that the passed atrributes in the to_guess
+        list can be guessed by the class
+
+        Parameters
+        ----------
+        to_guess: list of atrributes to be guessed then added to the universe
+        Returns
+        -------
+        True or False
+        """
+        for a in to_guess:
+            if a.lower() not in self._guess:
+                raise ValueError('{0} guesser can not guess the {1} atrribute'
+                                 .format(self.context, a))
+        return True
+
+    def guessTopologyAttribute(self, to_guess):
```

Comment:
In the absence of type hinting, could you please change the name of the `to_guess` variables to be clearer? here it's one string, but in `is_guessed` it's a list of strings -- that is quite confusing. Also, could you please name your methods with conventional snake_case and use TopologyAttr instead of TopologyAttribute?

Reply:
Do you plan to have an overall method for guessing all the attributes at once?

Reply:
Yes I'm planning to have a method for guessing all the attributes at once, this was just a test for guessing one attribute